internal/handlers/handlers.go

```diff
@@ -31,5 +36,8 @@ func (h *HandlerStr) InitHandlers() http.Handler {
 	// Content-Type: text/plain
 	mux.Post("/update/{metricType}/{metricName}/{metricValue}", h.handleUploads)
 
-	return mux
+	mux.Post("/update/", h.handleJSONUploads)
+	mux.Post("/value/", h.getJSONMetricValue)
+
+	return middleware.WithCompressedResponse(middleware.RequestLogger(mux, h.logger), h.logger)
```

Comment:
1. It is better to use `mux.Use` to register middlewares and then to return just `mux`
2. `Logger` should probably be the outmost midlleware (so it should be called before and after the compressing middleware)